app/components/Navigation.tsx

```diff
@@ -0,0 +1,101 @@
+import { memo, useState, Fragment, useMemo } from "react";
+import Home from "~/components/Home";
+import Profile from "~/components/Profile";
+import { useAuth0 } from "@auth0/auth0-react";
+import { Menu, Transition } from '@headlessui/react'
+
+//TODO: fix style of profile picture (maybe expand the image when the menu is opened as well)
+
+const Navigation = memo(function Nav({ apiBaseURL }: { apiBaseURL: string }){
```

Comment:
Could we do this with an `export default function` style component, for consistency?